test/sampletests/mixedFiles/sampleJs.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
```

Comment:
the parallel cli runner tests need to be updated also.

Reply:
done